docs/docs/cmd/spo/tenant/tenant-site-membership-list.mdx

```diff
@@ -0,0 +1,137 @@
+
+# spo tenant site membership list
+
+Retrieve information about default site groups' membership
+
+## Usage
+
+```sh
+m365 spo tenant site membership list [options]
+```
+
+## Options
+
+```md definition-list
+`-u, --siteUrl <siteUrl>`
+: The URL of the site.
+
+`-r, --role [role]`
+: Filter the results to include only users with the specified roles: `Owner`, `Member`, or `Visitor`.
+
+```
+
+<Global />
+
+## Remarks
+
+:::info
+
+To use this command, you must be a Global or SharePoint administrator. For other scenarios, refer to the `spo web get --withGroups` and `spo group member list` commands.
+
+:::
+
+## Examples
+
+Retrieves information about default site groups' owners, members, and visitors of the site.
+
+```sh
+m365 spo tenant site membership list --siteUrl https://contoso.sharepoint.com
+```
+
+Retrieves information about site owners.
+
+```sh
+m365 spo tenant site membership list --siteUrl https://contoso.sharepoint.com --role Owner
+```
+
+## Response
+
+<Tabs>
+  <TabItem value="JSON">
+
+  ```json
+  {
+  "AssociatedOwnerGroup": [
+    {
+      "email": "uName@contoso.onmicrosoft.com",
+      "loginName": "i:0#.f|membership|uName@contoso.onmicrosoft.com",
+      "name": "UserName",
+      "userPrincipalName": "uName@contoso.onmicrosoft.com"
+    }
+  ],
+  "AssociatedMemberGroup": [
+    {
+      "email": "internal@contoso.onmicrosoft.com",
+      "loginName": "i:0#.f|membership|internal@contoso.onmicrosoft.com",
+      "name": "internal",
+      "userPrincipalName": "internal@contoso.onmicrosoft.com"
+    }
+  ],
+  "AssociatedVisitorGroup": [
+    {
+      "email": "",
+      "loginName": "c:0-.f|rolemanager|spo-grid-all-users/dc109ffd-4298-487e-9cbc-6b9b1a2cd3e2",
+      "name": "Everyone except external users",
+      "userPrincipalName": null
+    }
+  ]
+}
+  ```
+  </TabItem>
+  <TabItem value="Text">
+
+  ```text
+  AssociatedMemberGroup : [{"email":"internal@contoso.onmicrosoft.com","loginName":"i:0#.f|membership|internal@contoso.onmicrosoft.com","name":"internal","userPrincipalName":"internal@contoso.onmicrosoft.com"}]
+  AssociatedOwnerGroup  : [{"email":"uName@contoso.onmicrosoft.com","loginName":"i:0#.f|membership|uName@contoso.onmicrosoft.com","name":"UserName","userPrincipalName":"uName@contoso.onmicrosoft.com"}]
+  AssociatedVisitorGroup: [{"email":"","loginName":"c:0-.f|rolemanager|spo-grid-all-users/dc109ffd-4298-487e-9cbc-6b9b1a2cd3e2","name":"Everyone except external users","userPrincipalName":null}]
+  ```
+
+  </TabItem>
+    <TabItem value="CSV">
+
+  email,loginName,name,userPrincipalName,associatedGroupType
+uName@contoso.onmicrosoft.com,i:0#.f|membership|uName@contoso.onmicrosoft.com,UserName,uName@contoso.onmicrosoft.com,Owner
+internal@contoso.onmicrosoft.com,i:0#.f|membership|internal@contoso.onmicrosoft.com,internal,internal@contoso.onmicrosoft.com,Member
+,c:0-.f|rolemanager|spo-grid-all-users/dc109ffd-4298-487e-9cbc-6b9b1a2cd3e2,Everyone except external users,,Visitor
```

Comment:
Mostly done, With the 3 items, I thought it would be good to show that we are presenting data for the Owner, Member, and Visitor groups by default. Bu of course I can still change it if needed.